.github/workflows/release-arm.yaml

```diff
@@ -87,3 +87,44 @@ jobs:
           sudo -E docker load -i image.tar
           sudo -E docker push "$IMAGE"
           sudo rm -rf build/IMAGE build/VERSION
+
+  image_and_iso_arm64_generic:
+    needs:
+      - get-matrix
```

Comment:
Since the matrix is defined below, I think you could get rid of this

Reply:
its mostly so it runs in parallel with the other jobs, in case the matrix is broken or something like that, it wont run the job unnecessarily  :)

Reply:
gotcha, makes sense